config/defaults.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  apiPrefix: "http://localhost:3000/api/v1"
+  apiPrefix: "http://localhost:5000/api/v1"
```

Comment:
I've chosen to use the `hero local web` command to start the BE server. This exposes the server on port 5000, not 3000 - although to be clear I actually don't understand why.

If you prefer not to use the heroic command for the test, we can move it back to 3000 and create a .js file in config listening on 5000.